pagopa-ecommerce-transactions-scheduler-service/transactions-scheduler-service.env

```diff
@@ -19,3 +19,13 @@ OTEL_LOGS_EXPORTER="none"
 OTEL_TRACES_SAMPLER="always_on"
 OTEL_EXPORTER_OTLP_HEADERS="Authorization=Bearer aaa"
 
+
+DEAD_LETTER_LISTENER_TRANSACTION_FIXED_DELAY_MILLIS=1000
+DEAD_LETTER_LISTENER_TRANSACTION_MAX_MESSAGE_PER_POLL=10
+DEAD_LETTER_LISTENER_TRANSACTION_QUEUE_NAME=pagopa-ecommerce-transactions-dead-letter-queue
+DEAD_LETTER_LISTENER_NOTIFICATION_FIXED_DELAY_MILLIS=1000
+DEAD_LETTER_LISTENER_NOTIFICATION_MAX_MESSAGE_PER_POLL=10
+DEAD_LETTER_LISTENER_NOTIFICATION_QUEUE_NAME=pagopa-ecommerce-notifications-service-errors-queue
+ECOMMERCE_STORAGE_DEAD_LETTER_QUEUE_ENDPOINT=http://storage:10001/devstoreaccount1
+ECOMMERCE_STORAGE_DEAD_LETTER_QUEUE_ACCOUNT_NAME=devstoreaccount1
+ECOMMERCE_STORAGE_DEAD_LETTER_QUEUE_KEY=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==
```

Comment:
Can we use a single connection string?

Reply:
I understand, I meant instead of 1 connection string + 4 parameters -> 2 connection strings :)